### ISSUE TITLE
Readme tweaks

### ISSUE THREAD

**Issue by craigfrancis** [staabm/phpstan-dba#70]
This is just my initial thoughts, feel free to ignore/edit as you see fit...

1) The README.md file currently has two cases of calling `QueryReflection::setupReflector()`

https://github.com/staabm/phpstan-dba/blob/main/README.md?plain=1#L46
https://github.com/staabm/phpstan-dba/blob/main/README.md?plain=1#L77

But this method requires a `RuntimeConfiguration`; for now I've simply used `new RuntimeConfiguration()`; and this requires `use staabm\PHPStanDba\QueryReflection\RuntimeConfiguration`.

2) I'd move the `Installation` section above `Usage`, as you would need to do this first.

3) Probably remove/rename the `Usage` heading, as this is part of `Installation` or could be called `Setup`.

4) I'd create the `phpstan-bootstrap.php` (named like this to match the example given in the phpstan documentation, and not conflict with any existing `bootstrap.php` files); and I'd do this as the first step after `composer require`, so the file exists before trying to explain that a file (of unknown content) needs to be added to the phpstan config file... you can also add a note that other bootstrap options are below, e.g. if you "don't want to depend on a database at PHPStan analysis time".

5) Then I'd do the `phpstan.neon` config update in a second (single) step, and use a basic example (because people are lazy)... saying something like "either create or update your `phpstan.neon` file so that it uses [bootstrapFiles](https://phpstan.org/config-reference#bootstrap) and it [includes dba.neon](https://phpstan.org/config-reference#multiple-files), e.g."

```
parameters:
  level: 9
  paths:
    - public
  bootstrapFiles:
    - phpstan-bootstrap.php

includes:
  - ./vendor/staabm/phpstan-dba/config/dba.neon
```

6) Then you could add a "using" step, e.g.

```
./vendor/bin/phpstan analyse -c phpstan.neon
```

**Comment by staabm** [comment]
I agree with all your points.

> 4\. I'd create the `phpstan-bootstrap.php` (named like this to match the example given in the phpstan documentation, and not conflict with any existing `bootstrap.php` files); and I'd do this as the first step after `composer require`, so the file exists before trying to explain that a file (of unknown content) needs to be added to the phpstan config file... you can also add a note that other bootstrap options are below, e.g. if you "don't want to depend on a database at PHPStan analysis time".

maybe we should even name this file `phpstan-dba-bootstrap.php`? 

are you willing to provide a PR with the suggested changes?

thx @craigfrancis

**Comment by craigfrancis** [comment]
Yep, I'll try to look at it tomorrow.

**Comment by craigfrancis** [comment]
Sorry for the delay, some work stuff cropped up.

I've created [Pull Request 107](https://github.com/staabm/phpstan-dba/pull/107), which includes the suggested file name `phpstan-dba-bootstrap.php`.

[Preview](https://github.com/staabm/phpstan-dba/blob/c5779d4757c454b72521450cecce8c806c045345/README.md).

Feel free to edit/delete as you prefer.

I did play with `<details>` show/hide element, but I though it made it a bit more complicated (as it needed to talk about `phpstan/extension-installer` and the `includes:` config, which I think are better included as part of the example).

Also, now that `RuntimeConfiguration` is required, I've updated the example bootstrap file to include it, with the commented out example on how to enable the new debug mode.

Hope that helps.
